qiskit/visualization/counts_visualization.py

```diff
@@ -64,7 +64,7 @@ def plot_histogram(
         color (list or str): String or list of strings for histogram bar colors.
         number_to_keep (int): The number of terms to plot and rest
             is made into a single bar called 'rest'.
-        sort (string): Could be 'asc', 'desc', or 'hamming'.
+        sort (string): Could be 'asc', 'desc', 'hamming', 'value', or 'value_desc'.
```

Comment:
does this need to say that default: `asc`?

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/6592/commits/47db559b49d8805beb418a2aaf1dcb2cd5f4c3e5